jbpm-benchmarks/kieserver-performance-tests-jmh/pom.xml

```diff
@@ -0,0 +1,282 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.jbpm</groupId>
+    <artifactId>jbpm-benchmarks</artifactId>
+    <version>7.60.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>kieserver-performance-tests-jmh</artifactId>
+
+  <name>KIE Server Performance Tests JMH</name>
+
+  <properties>
+    <kit.version>${version.org.kie}</kit.version>
+    <metrics.version>4.1.18</metrics.version>
+
+    <!-- >>>>>>>>>>>>>>>> Performance configuration <<<<<<<<<<<<<<<<<<<< -->
+
+    <projectName>BPM Suite</projectName>
+    <suite>LoadSuite</suite>
+    <scenario></scenario>
+    <org.kie.perf.suite.test-package></org.kie.perf.suite.test-package>
+    <startScriptLocation>./run.sh</startScriptLocation>
+
+    <!-- Duration (in seconds), Iteration -->
+    <runType>Iteration</runType>
+    <duration>30</duration>
+    <iterations>1</iterations>
+
+    <warmUp>false</warmUp>
+    <warmUpCount>3</warmUpCount>
+    <warmUpTime>30</warmUpTime>
+    <auditLogging>true</auditLogging>
+
+    <!-- Properties applicable to concurrent testing -->
+    <threads>5</threads>
+
+    <!-- Reporting - Console, CSVSingle, PerfRepo, CSV, (periodicity in seconds) -->
+
+    <reporterType>Console</reporterType>
+    <periodicity>5</periodicity>
+    <reportDataLocation>reportData</reportDataLocation>
```

Comment:
All these properties starting with `suite` are probably not needed anymore as the were used just in KPK, right?